source/agora/consensus/protocol/Nominator.d

```diff
@@ -1240,12 +1241,25 @@ extern(D):
                         msg));
 
                 Amount total_rate;
+
+                // If payout block we do not check fees for coinbase tx (which is last in set)
+                auto payout = slot_idx >= 2 * this.params.PayoutPeriod && slot_idx % this.params.PayoutPeriod == 0;
+
                 foreach (const ref tx_hash; data.tx_set)
                 {
+                    if (payout && tx_hash == data.tx_set[$ - 1])
+                        continue;   // This is the coinbase tx so has no fees
+
                     Amount rate;
+
                     auto errormsg = this.ledger.getTxFeeRate(tx_hash, rate);
                     if (errormsg == Ledger.InvalidConsensusDataReason.NotInPool)
-                        continue; // most likely a CoinBase Transaction
+                    {
+                        log.warn("combineCandidates: tx {} not in local pool so will skip this candidate",
+                            tx_hash);
+                        skip_candidate = true; // We will not include this candidate
+                        continue;
```

Comment:
I think we can just `break` here

Reply:
Yes I wasn't sure if `break` was better here or not. It is the last in the set so `continue` and `break` result in the same runtime behavior.